pkg/cmd/server/origin/master.go

```diff
@@ -274,14 +274,6 @@ func (c *MasterConfig) InstallProtectedAPI(container *restful.Container) []strin
 	legacyAPIVersions := []string{}
 	currentAPIVersions := []string{}
 
-	if configapi.HasOpenShiftAPILevel(c.Options, OpenShiftAPIV1Beta3) {
-		if err := c.api_v1beta3(storage).InstallREST(container); err != nil {
-			glog.Fatalf("Unable to initialize v1beta3 API: %v", err)
-		}
-		messages = append(messages, fmt.Sprintf("Started Origin API at %%s%s", OpenShiftAPIPrefixV1Beta3))
-		legacyAPIVersions = append(legacyAPIVersions, OpenShiftAPIV1Beta3)
-	}
-
```

Comment:
add a comment below explaining why we continue calling `initAPIVersionRoute(root, LegacyOpenShiftAPIPrefix, legacyAPIVersions...)` so no one tries to remove it